src/scripts/filter_rare_disease_list.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+import pandas as pd
+
+def main(args):
+    # Read the input files into pandas dataframes
+    old_df = pd.read_csv(args.old_rare_list, sep='\t')
+    new_df = pd.read_csv(args.new_rare_list, sep='\t')
+
+    # Filter the dataframes
+    df_added = old_df[~old_df['ID'].isin(new_df['ID'])]
```

Comment:
### Implementation
Implementation is fine to me. Really straightforward and quick.

---

I saw that I am set as a 'code owner'f or Python. @matentzn If you want me to review other stuff here in addition to Python files, let me know.

Reply:
@joeflack4 thanks for your review in general! I will assign you if I need a specific review, else I kinda like your eyes on things if you see it fly by, whenever you have time!